### PR TITLE
Add Python api docs for Auto ML and Chronos Forecasters

### PR DESCRIPTION
**Pull request by cabuliwallah**
- Add two sections **AutoML API** and **ChronosForecasters API** in  readthedocs
- Make sure all parameters and return values has a description
- Add a code example for each class
- Add an intro for each class

You can preview the doc [here](https://yuyin-analytics-zoo.readthedocs.io/en/latest/doc/PythonAPI/AutoML/automl.html)

### CODE REVIEW

**Comment by shane-huang** [comment]
@yushan111 please kindly review the docs for AutoEstimator at [here](https://yuyin-analytics-zoo.readthedocs.io/en/latest/doc/PythonAPI/AutoML/automl.html#orca-automl-auto-estimator)

**Comment by shane-huang** [comment]
please also fix the references to the API docs in Chronos usere guide at [here](https://yuyin-analytics-zoo.readthedocs.io/en/latest/doc/Chronos/Overview/chronos.html#create-a-forecaster), Specifically
* add all 5 forecasters in 4.2.1-4.2.5 (currently we have 3)
* all the references links to API docs in all the sections to the correct links to the API docs link you've just added. e.g. In sentence "View Network Traffic Prediction notebook and LSTMForecaster API Doc for more details.", the "LSTMForecaster API Doc" should point to this [link](https://yuyin-analytics-zoo.readthedocs.io/en/latest/doc/PythonAPI/Chronos/forecasters.html#chronos-model-forecast-lstm-forecaster)

**Comment by shane-huang** [comment]
LGTM

**Comment by yushan111** [comment]
> @yushan111 please kindly review the docs for AutoEstimator at [here](https://yuyin-analytics-zoo.readthedocs.io/en/latest/doc/PythonAPI/AutoML/automl.html#orca-automl-auto-estimator)

Shall we remove the docs in constructor since we don't hope user instantiate an `AutoEstimator` with a model_builder directly?

File: docs/readthedocs/source/doc/PythonAPI/Chronos/forecasters.rst [hunk 1]
```diff
@@ -0,0 +1,81 @@
+Chronos Forecasters API
+=====================
+
+chronos.model.forecast.lstm_forecaster
+----------------------------------------
+
+Long short-term memory(LSTM) is a special type of recurrent neural network(RNN). We implement the basic version of LSTM - VanillaLSTM for this forecaster for time-series forecasting task. It has two LSTM layers, two dropout layer and a dense layer. LSTMForecaster is derived from tfpark.KerasMode, and can use all methods of KerasModel. Refer to `tfpark.KerasModel API Doc <https://github.com/intel-analytics/analytics-zoo/blob/master/docs/docs/APIGuide/TFPark/model.md>`__ for details.
```
Comment (TheaperDeng) [inline docs/readthedocs/source/doc/PythonAPI/Chronos/forecasters.rst:7]: is tfpark doc our only choice for the forecasters inherit from tfparkforecaster? I think that it is deprecated? @yushan111

**Comment by cabuliwallah** [comment]
Jenkins passed:
http://10.239.176.111:18888/job/ZOO-PR-Validation/5700/

**Pull request merged**